packages/manager/apps/dedicated/package.json

```diff
@@ -36,6 +36,7 @@
     "@ovh-ux/manager-cookie-policy": "^0.0.0 || ^1.0.0",
     "@ovh-ux/manager-core": "^12.0.0 || ^13.0.0",
     "@ovh-ux/manager-dbaas-logs": "^0.0.0 || ^1.0.0",
+    "@ovh-ux/manager-dedicated-housing": "^0.0.0 || ^0.1.0-alpha",
```

Comment:
According to the commit type `refactor`, no bump will be made,  the version will stay `0.0.0`, maybe a `feat` will help releasing a `0.1.0-alpha`